napari/resources/qt.py

```diff
@@ -2,21 +2,262 @@
 
 # Resource object code
 #
-# Created: Sat Oct 5 21:19:34 2019
-#      by: The Resource Compiler for PySide2 (Qt v5.12.3)
+# Created: Sun Nov 24 10:05:59 2019
+#      by: The Resource Compiler for PySide2 (Qt v5.13.2)
 #
 # WARNING! All changes made in this file will be lost!
 
-from qtpy import QtCore
+from PySide2 import QtCore
```

Comment:
you need to manually change this to `qtpy` for cross compatibility

Reply:
ah!  thanks for catching that.  I'll add a test to make sure it gets caught in the future.